### DISCUSSION TITLE
Dynamic import with variable and alias path error not found

### DISCUSSION THREAD

**Discussion by dimaslanjaka** [webpack/webpack#17737] [Q&A]
My configs

alias config
```js
module.exports = {
  '@root': './'
};
```

working by using relative import
```js
const importPath = './posts/post-id/index.jsx';
const { default: Post } = await import(`${importPath}`);
```
working using alias import `@root/src`
```js
import { default as Post } from '@root/src/post-id/index'
```
but not working **dynamic import** using alias import `@root/src`
```js
const importPath = '@root/src/posts/post-id/index.jsx';
const { default: Post } = await import(`${importPath}`);
```

I've tried these but not works
- removing `.jsx` extension
- changing `@root` config to absolute path using `path.resolve`

My Configs
- [alias paths](https://github.com/dimaslanjaka/webpack-boilerplate/blob/f02cfd1/.alias.js)
- - this will imported from babel config and webpack common config
- [webpack common config](https://github.com/dimaslanjaka/webpack-boilerplate/blob/f02cfd1/config/webpack.common.js)
- [typescript config](https://github.com/dimaslanjaka/webpack-boilerplate/blob/f02cfd1/tsconfig.json)
- [babel config](https://github.com/dimaslanjaka/webpack-boilerplate/blob/f02cfd1/.babelrc.js)

dynamic import problem https://github.com/dimaslanjaka/webpack-boilerplate/blob/f02cfd1b893d0153223bfe594924e698284b7b16/src/routeMap.tsx#L6 which imported from https://github.com/dimaslanjaka/webpack-boilerplate/blob/f02cfd1b893d0153223bfe594924e698284b7b16/src/App.tsx#L6

I need working dynamic import with alias path to be used later in other page route, how to make **Dynamic import with variable and alias path** working?

**Comment by alexander-akait** [comment]
Yeah, the known problem, related https://github.com/webpack/webpack/issues/13865
